assets/styles/navbar.scss

```diff
@@ -38,6 +38,7 @@
     a {
         $duration: 0.5s;
         transition: $duration ease-out;
+        z-index:10;
```

Comment:
now is this really useful?

Reply:
I need a Z-index, indeed maybe not z-index 10